bundles/org.openhab.binding.yamahamusiccast/README.md

```diff
@@ -0,0 +1,162 @@
+# Yamaha MusicCast Binding
+
+Binding to control Yamaha models via their MusicCast protocol (aka Yamaha Extended Control).
+With support for 4 zones : main, zone2, zone3, zone4. Main is always present. Zone2, Zone3, Zone4 are read from the model.
+
+UDP events are captured to reflect changes in the binding for:
+
+- Power
+- Mute
+- Volume
+- Input
+- Presets
+- Sleep
+- Artist
+- Track
+- Album
+- Album Art
+- Repeat
+- Shuffle
+
+## Supported Things
+
+Each model (AV Receiver, ...) is a Thing. Things are linked to a Bridge for receiving UDP events.
+
+## Discovery
+
+No auto discovery
+
+## Binding Configuration
+
+N/A
+
+## Thing Configuration
+
+| Parameter              | type    | description                                             | Advanced | Required      |
+|------------------------|---------|---------------------------------------------------------|----------|---------------|
+| configHost             | String  | IP address of the Yamaha model (AVR, ...)               | false    | true          |
+| configSyncVolume       | Boolean | Sync volume across linked models (default=false)        | true     | false         |
+
+
+## Channels
+
+| channel               | type   | description                                                        |
+|-----------------------|--------|--------------------------------------------------------------------|
+| channelPower          | Switch | Power ON/OFF                                                       |
```

Comment:
Is there a reason why you prefixed the Channels with `channel`?

Reply:
This was done in the beginning for clarity. Is it best to remove the prefix? 
How about the prefix I used for config? Is that one OK then?

Reply:
As it is redundant it should be removed. This applies also to the config, if it is redundant.

Reply:
Done, new push has been done.